script.js

```diff
@@ -409,26 +429,33 @@ function handleData([
  *****************************************/
 
 function handleCitiesLayer(geojson) {
-  // styling for the cities layer: style cities conditionally according to a presence of a moratorium
+  // styling for the cities layer: style cities conditionally according to moratorium rating scale 1 to 3
   const pointToLayer = (feature, latlng) => {
-    // style cities based on whether their moratorium has passed
-    if (feature.properties.passed === "Yes") {
-      return L.circleMarker(latlng, {
-        color: "#4dac26",
-        fillColor: "#b8e186",
-        fillOpacity: fillOpacity,
-        radius: pointRadius,
-        weight: strokeWeight
-      });
-    } else {
-      return L.circleMarker(latlng, {
-        color: "#d01c8b",
-        fillColor: "#f1b6da",
-        fillOpacity: fillOpacity,
-        radius: pointRadius,
-        weight: strokeWeight
-      });
-    }
+    return L.circleMarker(latlng, {
+      color: strokeColorScale[feature.properties.scale],
```

Comment:
The new field with the numeric score will be called "range" so the correct property would be `feature.properties.range` once the data is in.